sound/soc/codecs/Kconfig

```diff
@@ -842,7 +842,6 @@ config SND_SOC_DA9055
 
 config SND_SOC_DMIC
 	tristate "Generic Digital Microphone CODEC"
-	depends on GPIOLIB
```

Comment:
The commit message was saying "Remove spurious gpiolib select" and in Kconfig it is `depends`?

Reply:
yes, will fix.

Reply:
actually can't fix, the patch is from Mark Brown and taken from alsa-devel :-)